documentation/functional.md

```diff
@@ -0,0 +1,110 @@
+Findex functional documentation
+===============================
+
+## How to securely manage an index
+
+This documents describes in details the functionalities of Findex.
+
+The goal of Findex is to allow users to securely manage an index. To know more
+about the security guarantees of Findex, see the [security][sec_doc]
+documentation.
+
+### An index as a graph
+
+An index can be thought of as a graph: each indexed keyword is associated a
+node, which contains all data indexed under this keyword; some of this data can
+be other keywords which can be represented as directed edges linking the
+vertices of each one of the associated keywords.
```

Comment:
what else can it be ?

Reply:
It can also be viewed as a special type of data, as it is in the MM model.